kafka/strimzi/README.md

```diff
@@ -1,19 +1,19 @@
 # Kafka Integration
 
-Seldon Core v2 requies Kafka to implement datacentric inference Pipleines. 
+Seldon Core v2 requires Kafka to implement Data-Centric inference Pipelines.
```

Comment:
```suggestion
Seldon Core v2 requires Kafka to implement data-centric inference Pipelines.
```
:thought_balloon: Data-centricity is a concept, but not a proper noun as if it were a company or a concrete list of rules to follow